doc/lua_api.txt

```diff
@@ -3916,6 +3916,13 @@ The Biome API is still in an experimental phase and subject to change.
     -- ^ Minimum and maximum `y` positions these decorations can be generated at.
     -- ^ This parameter refers to the `y` position of the decoration base, so
     --   the actual maximum height would be `height_max + size.Y`.
+        spawn_by = "default:water",
+    --  ^ Node that the decoration only spawns next to.
+    --  ^ The neighbours checked are the 16 nodes level with and surrounding
+    --  ^ these 2 nodes: the surface node and the node above the surface node.
```

Comment:
Zeno was right, the original wording was confusing.  However I'm not too sure this is an improvement.....

We should standardize on some phrasing here:
Instead of "horizontally surrounding" what about "node neighbors, including diagonal nodes, in the X and Z directions"?

So the wording would be more like this:
"Two planes of neighboring nodes (including diagonal neighbors), one at Y = surface and Y = surface - 1" ?  Is this an improvement?

Which brings up my next question:  Is it correct to leave a hole in the second lower plane when counting neighbors?  Shouldn't you be checking the lower center node (that is, offset (0, -1, 0))?